scss/components/_reader.scss

```diff
@@ -2,4 +2,11 @@ menupopup.tags-popup {
 	font: inherit;
 	min-width: 300px;
 	padding-inline: 8px;
+
+	@media (-moz-platform: windows) {
```

Comment:
I am conflicted about the use of `-moz-platform`. Generally, it's probably the more convenient way to do platform-specific styling and I would prefer it in place of generic style sheets with platform specific modifications in a single file. However, we are already doing platform-specific stylesheets. We should either decide on one or the other approach and stick to it, otherwise it becomes complicated and annoying finding platform-specific styling. I understand this gets somewhat even more muddled when copying styles from Mozilla's own stylesheets.

Reply:
Yeah, here is just a quick fix for the tag box popup on Windows, which is never right since the redesign. Actually we should find out what makes the padding not work on Windows and do the fix there.

Why we use the `menupopup` to contain a `tags-box` CE, instead of a `panel`?